sklearn/utils/estimator_checks.py

```diff
@@ -1640,7 +1674,7 @@ def check_class_weight_balanced_classifiers(name, classifier_orig, X_train,
 
 @ignore_warnings(category=(DeprecationWarning, FutureWarning))
 def check_class_weight_balanced_linear_classifier(name, Classifier):
-    """Test class weights with non-contiguous class labels."""
+    """Test class weights with non - contiguous class labels."""
```

Comment:
undo this change, please

Reply:
@jnothman nope just stuck on old python issue last week, this week I had continuous exams( so wasn't online most of the time)

I have my last exam on Sunday (will finish by Monday, it's mostly done)

Reply:
@jnothman I will undo the change

plus @jnothman I checked the python2.7 issue and I found out the issue with scipy

if we call asformat function with int64 indices based sparse matrix, it crashes

Reply:
@jnothman I am forcing the indices to be 64 bits(might be an issue due to this only,
because 64bit indices are supported in python2.7 by scipy)